app/scripts/components/date/from-now.filter.js

```diff
@@ -5,6 +5,8 @@
 
 angular.module("chuvApp.components.filters", []).filter("fromNow", function() {
   return function(date) {
-    return moment(date).fromNow();
+    var dateFromServer = moment.utc(date).toISOString();
+    var newDateInUTC = moment().utc().toISOString();
+    return moment( dateFromServer ).from( moment(newDateInUTC) );
   };
```

Comment:
Nice, thanks. But ideally it should be be in a separate branch. ( Even though I have to recognize that our side could express more rigor about the process. Hopefully it will be improved in the near future. )